rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMProxyUserInterface.java

```diff
@@ -112,6 +112,11 @@ public BooleanWrapper addNode(String arg0, String arg1) {
         return target.addNode(arg0, arg1);
     }
 
+    @Override
+    public BooleanWrapper acquireNodes(String arg0, int arg1, Map<String, ?> arg2) {
+        return target.acquireNodes(arg0, arg1, arg2);
```

Comment:
Having more self-expressive variable name than arg0, arg1 and arg2 could be an improvement.